services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeRepositoryITests.kt

```diff
@@ -26,6 +32,7 @@ constructor(val entityManager: TestEntityManager, val hankeRepository: HankeRepo
     }
 
     @Test
+    @Transactional
```

Comment:
Why `@Transactional`?

Reply:
I guess it is not needed. The previous implementation of this class was a bit different (e.g. it used h2 as a database generated from our jpa entities). As the entitymanager here might be redundant, I'll do some refactoring.

Reply:
Fixed: removed entitymanager and took out transactional annotation where I could. Some tests still need it as they fetch lazy initialized fields (requires transaction).

Reply:
I never realized this class used H2 as the database. That explains why there's an H2 dependency in `build.gradle.kts`. Can that be removed now? I don't think any other tests use it. At least, there are no more tests with the `@DataJpaTest` annotation.

Reply:
Removed.